src/routes/(console)/organization-[organization]/change-plan/+page.svelte

```diff
@@ -114,6 +115,10 @@
                 null
             );
 
+            const paidInvoices = await sdk.forConsole.billing.listInvoices(data.organization.$id, [
```

Comment:
this and the fetch below can be extracted into a separate function and not `await` it here. both API calls not really console functionality and can be loaded in the background.